spec/features/stats_page_spec.rb

```diff
@@ -1,11 +0,0 @@
-require 'spec_helper'
-require 'rails_helper'
-
-describe "Stats page", type: :feature, :vcr => {:cassette_name => "place" } do
-
-  it "should have a highchart", js: true do
-    visit "/stats"
-    expect(page).to have_css(".highcharts-container")
-  end
-
```

Comment:
hmmmm I think we were going to keep the stats page (bc it's interesting & could be used later at some point). Can you leave this spec in here? Or was it somehow messing stuff up?